src/main/webapp/recipe-display.html

```diff
@@ -149,15 +152,92 @@
         })
       }
 
+      /* Event triggered by clicking on an Edit button.
+       * Adds form to the comment for user to specify edits
+       */
+      function editComment(commentEl) {
+        commentEl.removeChild(commentEl.querySelector("hr"))
+        commentEl.appendChild(editForm)
+        commentEl.appendChild(submitEdit)
+        commentEl.appendChild(document.createElement("hr"))
+
+        editForm.value = commentEl.querySelector("p").innerText
+        submitEdit.innerText = "Submit"
+        submitEdit.addEventListener("click", submitEditEvent)
+      }
+
+      /* Event triggered by clicking Submit Button under Edit
+       * Removes edit form, calls function to upload changes to DB
+       */
+      function submitEditEvent(e) {
```

Comment:
Maybe should add a "cancel" button to cancel the action and then refactor this function to switch between cancel and submit logic? Not necessary but could improve the user experience!